openjdk/ProblemList_openjdk11-openj9.txt

```diff
@@ -261,6 +261,7 @@ java/util/stream/boottest/java.base/java/util/stream/NodeTest.java https://githu
 java/util/stream/test/org/openjdk/tests/java/util/SplittableRandomTest.java 	https://github.com/eclipse/openj9/issues/4613 	generic-all
 java/util/stream/test/org/openjdk/tests/java/util/stream/CountLargeTest.java	https://github.com/eclipse/openj9/issues/9040	linux-aarch64
 java/util/stream/test/org/openjdk/tests/java/util/stream/ToArrayOpTest.java	https://github.com/eclipse/openj9/issues/3447	generic-all
+java/util/WeakHashMap/GCDuringIteration.java	https://github.com/eclipse/openj9/issues/10256	generic-all
```

Comment:
Can you change the issue to point to https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297 (a placeholder for permanent excludes, openjdk tests that are not and will never be applicable to openj9) for the 3 entries in this PR?  I will reference openj9/10256 in 1297.

When we add automation to reinclude tests by checking if issues in problemlists are closed, issue 1297 will be exempt, but other issues not.

Reply:
Updated.